README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My complete name is *Miguel Ángel Rosales Legorreta*
+[My linkedin profile](www.linkedin.com)
```

Comment:
Great use of a link! Unfortunately, it doesn't work for me in the rendered version of your file, though the syntax is correct. I assume it may have something to do with spacing